### ISSUE TITLE
DO statement should be on a separate line: add exception for empty block

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#1057]
Sometimes we override a trigger. To avoid the empty block rule they include a comment.
````
ON WRITE OF <table> OVERRIDE DO: /* nothing */ END.
````
Since 2.21 this "block" is being flagged by the statement should be on a separate line rule. 

The following is **also** being flagged:
````
on delete of <table> override do:
   /* some reason */
end.
````
Can this case be suppressed without having to add a proparse directive?

**Comment by gquerret** [comment]
Yes, second example (`on delete...`) reports an issue which is clearly a FP.

**Comment by gquerret** [comment]
Fixed in main
